maintainers/maintainer-list.nix

```diff
@@ -4911,6 +4911,12 @@
     github = "damhiya";
     githubId = 13533446;
   };
+  damidoug = {
+    email = "contact@damidoug.dev";
+    github = "damidoug";
+    githubId = 75175586;
+    name = "Douglas Damiano";
+  };
```

Comment:
You added yourself to the maintainer list, but did not add yourself as a maintainer to any package. Was this a mistake?

Reply:
I have an PR (#368618) to maintainer-list.nix but it was rejected, @drupol told me first to apply pr on pkgs.